src/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>9.0.1-preview</VersionPrefix>
+    <VersionPrefix>9.0.2-preview</VersionPrefix>
```

Comment:
@Shane32  I would like to switch to MinVer here like I suggested in https://github.com/graphql-dotnet/graphql-dotnet/pull/3562 so we can see how this looks and feels.